jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -29,16 +29,18 @@ class MRSIMatchStep(Step):
         # General sky matching parameters:
         bkg_degree = integer(min=0, default=1) # Degree of the polynomial for background fitting
         subtract = boolean(default=False) # subtract computed sky from 'images' cube data?
-
+        skip = boolean(default=True) # Step must be turned on by parameter reference or user
     """
 
     reference_file_types = []
 
     def process(self, images):
-        all_models2d = ModelContainer(images)
 
-        chm = {}
+        # Provide warning to user this code is deprecated
+        warnings.warn("mrs_imatch deprecated", RuntimeWarning)
```

Comment:
I'm not sure this needs to be a software warning - maybe just log a warning instead, for user information?

Reply:
changed to log.warning if run